docs/xks/developer-guide/ci-cd/flux.md

```diff
@@ -117,3 +117,32 @@ NAME       READY   STATUS
 apps-dev   True    Applied revision: main/9baa401630894b78ecc5fa5ebdf72c978583dea8   47h
 tenant1    True    Applied revision: main/9baa401630894b78ecc5fa5ebdf72c978583dea8   2d2h
 ```
+
+## Helm
+
+You can of course use flux to manage your helm charts.
+
+### Upgrade retries exhausted
+
+For different reasons come in to a state of no more tries to apply your helm release happens and the error shown is `Upgrade retries exhausted`.
```

Comment:
For different reasons, the helm release can come into a state of no more retries to apply your helm release. When that happens, the error shown is `Upgrade retries exhausted`.

?